### PR TITLE
Reworked README.md file; added writing-style-guidelines.md

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
### Check List

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by rcv-aws — DISMISSED** [review]

**Review by 3manuek — APPROVED** [review]

**Review by rcv-aws — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -51,43 +38,138 @@ $ bundle install
...
+    * execute → use start or run 
+    * abort → use stop
+    * enable → use turn on, activate, start
+    * disable → use turn off, deactivate, stop
+    * kill → use end or stop
+    * blacklist → use deny list
+    * master → use primary, main, or leader
+    * hang → use stop responding
+    * slave → use secondary, replica, or standby
+    * black day → use blocked day
```
Comment (nasbyj) [inline README.md:100]: Needs to be removed; blocked days only make sense internally.

File: _docs/writing-style-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (nasbyj) [inline _docs/writing-style-guidelines.md:1]: We shouldn't list all of this twice. It should either be in the main readme or here, but not both.

**Review by nasbyj — CHANGES_REQUESTED** [review]

File: _docs/writing-style-guidelines.md (hunk 2 above)
Comment (susanmdouglas-aws) [reply _docs/writing-style-guidelines.md:1]: Are you proposing we shouldn't keep this file in the main website content?  My concern with having it only in the README is that some contributors might not read the readme file...

Comment (susanmdouglas-aws) [reply _docs/writing-style-guidelines.md:1]: I would not expect this content to change often...

File: README.md (hunk 1 above)
Comment (susanmdouglas-aws) [reply README.md]: Update committed: 904dbb0..d57fd49  readme_reworked -> readme_reworked

**Review by 3manuek — APPROVED** [review]

**Review by gopinathpai — APPROVED** [review]

**Review by nasbyj — APPROVED** [review]

**Pull request merged**